lib/aida/front_desk.ex

```diff
@@ -72,4 +77,4 @@ defmodule Aida.FrontDesk do
   defp log_usage(bot_id, session_id) do
     SkillUsage.log_skill_usage(bot_id, "front_desk", session_id)
   end
-end
+end
```

Comment:
Keep trailing whitespace at end. Setup your editor to enforce it.

Reply:
On VSCode, change both the `files.insertFinalNewline` & `files.trimFinalNewlines` settings to `true`. And `files.trimTrailingWhitespace` for the whitespaces.